package.json

```diff
@@ -33,14 +33,15 @@
     "build": "npm run build:js && npm run build:types",
     "build:js": "babel src -d lib -x \".ts,.tsx\" --ignore '**/*.test.ts,**/*.test.tsx'",
     "build:types": "tsc --emitDeclarationOnly",
-    "format": "eslint src --ext \".ts,.tsx\" --fix",
+    "lint": "eslint src --ext \".ts,.tsx\" --fix",
+    "format": "prettier --write ./src/**/*",
     "prebuild": "npm run format",
     "pretest": "npm run build",
     "test": "jest --coverage"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "npm run test"
+      "pre-commit": "npm run format && npm run lint && npm run test"
```

Comment:
Just a suggestion. We could use the pre-commit hook to format and lint the code also.

Reply:
I was leveraging pre scripts to accomplish this -> `pretest` is run before `test`, and triggers `build`. `prebuild` is run before `build`, and triggers `format`. `format` triggers eslint, which is configured (.eslintrc) to run eslint and prettier rules

Reply:
Ohhh I see Nolan, it would end up doing the four things after all  (format, lint, test and build). Cool.